data-processing/tests/test_expand_macros.py

```diff
@@ -0,0 +1,247 @@
+from typing import List
+
+from common.expand_macros import Expansion, detect_expansions, expand_macros
+
+
+def to_bytes(*lines: List[str]) -> bytes:
+    return "\n".join(lines).encode()
+
+
+def test_detect_simple_macro():
```

Comment:
yay tests! ❤️

Reply:
:-) 

A haiku
Unit tests are like
hard timber walls of a house
on a blust'ry day.

Reply:
Haikus are easy
But sometimes they don't make sense
Refrigerator

Reply:
👏 😆